src/locales/tr.json

```diff
@@ -231,6 +231,38 @@
     "whyWeDoItHeader": "Neden yapıyoruz",
-    "whyWeDoItText": "İnsanların doğayı ve özelliklerini korumak gibi ahlaki bir yükümlülüğü vardır, çünkü çevreyi korumak insanlığı toksik maddelerden ve gıda kalitemizi korur. MENA'daki yeni neslin, yerel ve küresel ölçekte insani ve sosyal konulara inanmak ve fayda sağlamak için hareket etmek için doğru araçlarla desteklenmesi gerektiğine inanıyoruz. Doğru araçlar verildiğinde, bir nesil aktivist yetiştirmek imkansız değildir."
+    "whyWeDoItText": "İnsanların doğayı ve özelliklerini korumak gibi ahlaki bir yükümlülüğü vardır, çünkü çevreyi korumak insanlığı toksik maddelerden ve gıda kalitemizi korur. MENA'daki yeni neslin, yerel ve küresel ölçekte insani ve sosyal konulara inanmak ve fayda sağlamak için hareket etmek için doğru araçlarla desteklenmesi gerektiğine inanıyoruz. Doğru araçlar verildiğinde, bir nesil aktivist yetiştirmek imkansız değildir.",
+    "teamInfo": {
+      "ammar": {
+        "name": "Ammar Sammour",
+        "bio": "",
+        "role": "Ekip Lideri"
+      },
+      "ibrahim": {
+        "name": "Ibrahim Aljarad",
+        "bio": "Hey, ben Suriye'den İbrahim, Flatiron kodlama okulunda kodlama okudum ve Alfurat üniversitesinde ekonomi okudum.Re:coded organizasyonla bir ön uç web geliştirme temel eğitim programına katıldım ve oradan mezun oldum. Kodlama dışındaki diğer deneyimlerim, saha yapım ekiplerine teknik destek sağlayan uydu yapımcısı olarak çeşitli kanallar ve ajanslarla çalıştım.",
+        "role": "Web Arayüz Geliştirici"
+      },
+      "majd": {
+        "name": "Majd Ajroudi",
+        "bio": "Çok yönlü becerilere, çeşitli mühendislik alanlarında deneyime ve sosyal sorunları çözmek için teknolojiyi kullanma konusunda istekli bir genç geliştirici.",
+        "role": "Web Arayüz Geliştirici"
+      },
+      "sule": {
+        "name": "Sule Sinikoglu",
+        "bio": "Küresel Zorluklar üzerinde olumlu etki yaratmak için 'iyi kod' yazan bir Web Geliştiricisi.",
+        "role": "Web Arayüz Geliştirici"
+      },
+      "tarkan": {
+        "name": "Tarkan Mansuroglu",
+        "bio": "Re:coded Ön Uç Web Geliştirme Eğitim Kampı eğitimli ve kısmen kendi kendine öğretilen bir ön uç web geliştiricisi. Aynı zamanda tutkulu bir öğrenci, bilgisayar meraklısı, oyun tutkunu ve spor tutkunu. Becerilerini test etmek ve geliştirmek için gerçek hayattaki sorunları çözmeyi seviyor. Onu sosyal medyadan takip edin!",
+        "role": "Web Arayüz Geliştirici"
+      },
+      "yaman": {
+        "name": "Mohamad Yaman Sammani",
+        "bio": "Yaman, Kasım 2020'de Re: Coded Bootcamp'tan mezun olan genç bir Front-End web geliştiricisidir. Tutkusu, ev sahibi ülkelerdeki hizmet göçmenlerine kolay erişimi destekleyen programlar tasarlamaktır.",
```

Comment:
You may update like 
"Yaman, Kasım 2020'de Re: Coded Bootcamp'tan mezun olan Web Arayüz geliştiricisidir. Tutkusu, göçmenlere ev sahibi ülkelerdeki hizmetlere kolay erişimlerini sağlayan programlar tasarlamaktır."

Reply:
PTAL